en/getting_started/README.md

```diff
@@ -86,7 +87,7 @@ To install Qt:
   - **OSX:** Desktop Qt {{ book.qt_version }} clang 64 bit
     > **Note** iOS builds must be built using [XCode](http://doc.qt.io/qt-5/ios-support.html).
   - **Ubuntu:** Desktop Qt {{ book.qt_version }} GCC 64bit
-  - **Windows:** Desktop Qt {{ book.qt_version }} MSVC2015 **32bit**
+  - **Windows:** Desktop Qt {{ book.qt_version }} MSVC2018 **64bit**
```

Comment:
This is wrong isn't it?

Reply:
I changed to MSVC2017, which I hope is the change you expected?

Reply:
Yup. That should do it. I assume anything else missing we'll here from folks complaining.